treemap/script.js

```diff
@@ -243,55 +235,37 @@ d3.json('data.json', function(err, dataset) {
       return 'translate(' + x + ',' + y + ')';
     });
 
-  var introy = config.width < 400 ? -20 : 10;
   playerInfo
     .append('text')
-    .attr('class', function() {
-      if (config.width < 400) {
-        return 'playerInfo mobile';
-      } else {
-        return 'playerInfo';
-      }
+    .at({
+      class: 'playerInfo',
+      x: 0,
+      y: 10,
     })
-    .attr('x', 0)
-    .attr('y', introy)
     .tspans(function() {
```

Comment:
Does this need to be a function any more (not sure what `.tspans` requires)?

If so, we can make it an arrow function that immediately returns

Reply:
fixed but doesn't show